docs/source/01_tutorials/02_dataframe_model.md

```diff
@@ -0,0 +1,40 @@
+# DataFrameModel
+
+Alternatively, `pandera` support a class-based `DataFrameModel` schema.
+
+An example of schema look like:
+```python
+import pandera as pa
+from pandera.typing import Series
+
+
+class ExampleIrisDataSchema(pa.DataFrameModel):
+    sepal_length: Series[float] = pa.Field(gt=2000)
+```
+
+You should put this in `src/<kedro_pandera_tutorial/schema/example_iris_data.py` and create a `src/kedro_pandera_tutorial/schema/example_iris_data/__init__.py` file.
```

Comment:
Maybe we can *suggest* having these files but your code works without these naming conventions, doesn't it?

Reply:
It's not mandatory, but may make more sense if we implement the infer CLI for data model, as you will need a default name.

Reply:
Yes sure. I sugest we go with ``.../schema**s**/...`` with a plural for the folder name.